### ISSUE TITLE
Impossible to run simple example...

### ISSUE THREAD

**Issue by matdrapeau** [bprinty/Flask-Execute#22]
After spending some time to debug, it is impossible to run a simple flask app:

```
from flask import Flask
from flask_execute import Celery

celery = Celery()
app = Flask(__name__)
celery.init_app(app)

@app.route('/')
def hello_world():
    return 'Hello World!'


def add(x, y):
  return x + y

future = celery.submit(add, 1, 2)
future.result(timeout=1)

if __name__ == '__main__':
    app.run()
```

I get this error:
`subprocess.TimeoutExpired: Command '['celery', '-A', 'flask_execute.ext.celery', 'status']' timed out after 5 seconds`
where it looks like this line: https://github.com/bprinty/Flask-Execute/blob/dc14c7ec5e3ec874b608141402f344666246c453/flask_execute/plugin.py#L252
will raise a subprocess.TimeoutException and never been able to subsequently run a worker.

And then when trying to run worker directly using cli (`celery -A flask_execute.ext.celery worker`) that raises this exception still...
```
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask_execute/plugin.py", line 384, in submit
    return self.apply(func, args=args, kwargs=kwargs, **options)
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask_execute/plugin.py", line 401, in apply
    self.start()
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask_execute/plugin.py", line 344, in start
    if self.status():
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask_execute/plugin.py", line 455, in status
    output = cli.output('status', timeout=timeout)
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask_execute/cli.py", line 49, in output
    return subprocess.check_output(args, stderr=stderr, env=os.environ.copy(), **kwargs).decode('utf-8')
  File "/usr/lib/python3.8/subprocess.py", line 411, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib/python3.8/subprocess.py", line 491, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/usr/lib/python3.8/subprocess.py", line 1024, in communicate
    stdout, stderr = self._communicate(input, endtime, timeout)
  File "/usr/lib/python3.8/subprocess.py", line 1867, in _communicate
    self._check_timeout(endtime, orig_timeout, stdout, stderr)
  File "/usr/lib/python3.8/subprocess.py", line 1068, in _check_timeout
    raise TimeoutExpired(
subprocess.TimeoutExpired: Command '['celery', '-A', 'flask_execute.ext.celery', 'status']' timed out after 5 seconds
```

Running on python3.8

**Comment by matdrapeau** [comment]
If I move the task submission into routes, it seems to resolve the initialisation of the worker.

But here's another app example which still raises another exception
```
from flask import Flask
from flask_execute import Celery

app = Flask(__name__)
celery = Celery()
celery.init_app(app)

def test_add(x, y):
    return x + y

@app.route('/')
def hello_world():
    future = celery.submit(test_add, 1, 2)
    return 'result: {}'.format(future.result(timeout=1))

if __name__ == '__main__':
    app.run()
```

Exception:
```
Traceback (most recent call last):
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/m/flask_celery_test/app.py", line 19, in hello_world
    return 'result: {}'.format(future.result(timeout=1))
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/flask_execute/futures.py", line 43, in result
    self.__proxy__.wait(timeout=timeout)
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/celery/result.py", line 223, in get
    return self.backend.wait_for_pending(
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/celery/backends/asynchronous.py", line 201, in wait_for_pending
    return result.maybe_throw(callback=callback, propagate=propagate)
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/celery/result.py", line 335, in maybe_throw
    self.throw(value, self._to_remote_traceback(tb))
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/celery/result.py", line 328, in throw
    self.on_ready.throw(*args, **kwargs)
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/vine/promises.py", line 234, in throw
    reraise(type(exc), exc, tb)
  File "/home/m/flask_celery_test/venv/lib/python3.8/site-packages/vine/utils.py", line 30, in reraise
    raise value
AttributeError: 'NoneType' object has no attribute 'Model'
```

**Comment by bprinty** [comment]
Thanks for filing the issue! This was caused by one of the functions that re-queries SQLAlchemy models for safer threaded operations. I updated the logic in that method to break if the project isn't using an ORM for managing database models.

See `Flask-Execute==0.1.6` for a version with those fixes. I'll also update the documentation for the project because the minimal application example isn't completely clear.
